### PR TITLE
Fix clang format current master

### PR DESCRIPTION
**Pull request by vhvb1989**

### CODE REVIEW

File: sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp [hunk 1]
```diff
@@ -2738,8 +2738,8 @@ namespace Azure { namespace Storage { namespace Blobs {
             Storage::Details::XmlWriter& writer,
             const SetServicePropertiesOptions& options)
         {
-          writer.Write(Storage::Details::XmlNode{
-              Storage::Details::XmlNodeType::StartTag, "StorageServiceProperties"});
+          writer.Write(Storage::Details::XmlNode{Storage::Details::XmlNodeType::StartTag,
+                                                 "StorageServiceProperties"});
           BlobServicePropertiesToXml(writer, options.Properties);
```
Comment (Jinming-Hu) [inline sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp:2743]: I used clang-format 11 to format this code. How come we got different result?

Comment (vhvb1989) [reply sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp:2743]: Was that on Windows?  
I can try to reproduce. Please share your env details and where did you install clang-format from.

If there's a difference from clang-format 9 vs 11, we will need to figure it out

Comment (Jinming-Hu) [reply sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp:2743]: I feel there's something wrong with my clang-format tool. please give me some time to investigate.

Comment (Jinming-Hu) [reply sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp:2743]: This is what I got from clang-format 9
image

This is what I got from clang-format 11

image


The result is the same as 11.0 for the built-in one of Visual Studio.

Comment (Jinming-Hu) [reply sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp:2743]: I think we should use the latest version or the built-in one of Visual Studio

Comment (vhvb1989) [reply sdk/storage/azure-storage-blobs/inc/azure/storage/blobs/protocol/blob_rest_client.hpp:2743]: I have created PRs for clang format 9, 10 and 11:
#1333, #1334 , #1335

We need to review and decide which one we want

**Pull request closed without merging**